week2/README.md

```diff
@@ -0,0 +1,88 @@
+#### Part 1 - Short-answer questions.
+a) Describe the two size classes in iOS.</br>
+**There are two size classes, *regular* and *compact*. *Compact* refers to a smaller than normal amount of space; for example, the top and bottom edge of Portrait mode for iPhones have a size class of "compact width." 
+
+***Regular* refers to a standard amount of space; the left and right edges of iPhones in Portrait mode have a "regular height." In Landscape mode, larger iPhone models have a "regular width," while smaller models have a "compact width"; all models of iPhones have a compact height in Landscape mode.**
+
+b) What is Continuous Learning, and why is it important in mobile development?</br>
+**Continuous learning is a way of keeping your skills sharp by always looking to improve on your knowledge and regularly using the skills you have in your chosen field. Because technology is always changing with the release of new devices, new operating systems and even new languages and software platforms, it's important to stay on top of the latest developments and best practices for the programming languages you use, and also stay fresh with the basics. This means practicing your coding skills regularly, compiling a list of resources and cheat sheets, and checking out blogs, books and courses on emerging changes for your field.**
+
+c) How can you find out what modifiers a View has?</br>
```

Comment:
1. Modifiers use “.” before calling it in the view. Once you type “.” it shows the modifiers that it supports.
2. The Modifiers Library
3. Attribute Inspector Add Modifier. Preview must be in Selectable Mode.

Reply:
Definitely forgot about those other two methods—I don't use the library enough!